ui/apps/platform/src/Containers/Integrations/utils/tableColumnDescriptor.ts

```diff
@@ -294,6 +300,12 @@ const tableColumnDescriptor: Readonly<IntegrationTableColumnDescriptorMap> = {
             },
         ],
     },
+    cloudSources: {
+        paladinCloud: [
```

Comment:
@ivan-degtiarenko my only question is whether `paladinCloud` key needs to be `TYPE_PALADIN_CLOUD` as in service response and like `type: 'TYPE_PALADIN_CLOUD'` you wrote in `defaultValues` of PaladinCloudIntegrationForm:
* table column descriptors here
* integration tile descriptor above in integrationsList.ts file (for this, I think this `'paladinCloud'` is okay because CloudSourceIntegrationSection renders the tile element directly instead of via descriptor)
* integration form map above in IntegrationForm.tsx file

Can you experiment to create an integration and then click the tile to display the cloud source integration table?

If it helps, I am available for a call Friday morning US Eastern time.

Reply:
Okay, after a good night of sleep, rereading code, and a call to discuss, we concluded that this is okay for MVP because there is only one cloud source integration.

We will need to think what to do about the difference between `paladinCloud` which is good for page address
/main/integrations/cloudSources/paladinCloud

but `type: 'TYPE_PALADIN_CLOUD'` which is in response whenever we need to filter different integrations in useIntegrations.ts as for example:
`return notifiers.filter(typeLowerMatches)`